pages/whats-new/tinacloud.tsx

```diff
@@ -0,0 +1,59 @@
+import { Layout } from 'components/layout'
+import client from 'tina/__generated__/client'
+import { TinaMarkdown } from 'tinacms/dist/rich-text'
+import { versionComponents } from 'components/styles/WhatsNewVisualComponents'
```

Comment:
Confusing naming - what are version components?
```suggestion
import { visualComponents } from 'components/styles/WhatsNewVisualComponents'
```